src/receive_imf.rs

```diff
@@ -114,7 +114,8 @@ enum ChatAssignment {
 
     /// Group chat without a Group ID.
     ///
-    /// This is not encrypted.
+    /// This is either encrypted or unencrypted. Encryption status never changes once the group is
```

Comment:
This comment lies. Encryption never downgrades, but it can upgrade if you remove the last address-contact from the group. I think it's fine.